rero_ils/modules/operation_logs/jsonschemas/operation_logs/operation_log-v0.0.1.json

```
@@ -65,21 +86,79 @@
       "type": "string",
       "enum": [
         "create",
-        "update"
+        "update",
+        "delete"
       ]
     },
+    "ill_request": {
+      "title": "ILL Request status",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "status"
+      ],
+      "properties": {
+        "status": {
+          "type": "string",
+          "title": "Status",
```

```suggestion
          "title": "Status",
          "type": "string",
```
https://github.com/rero/developer-resources/blob/master/data/jsonschema.md#order
Title, then type.